editor/components/FatalityVictimsCard.tsx

```diff
@@ -0,0 +1,97 @@
+import { Crash } from "@/types/crashes";
+import { Card, ListGroup, ListGroupItem } from "react-bootstrap";
+import { getInjuryColorClass } from "@/utils/people";
+
+interface FatalityVictimsCardProps {
+  crash: Crash;
+}
+
+export default function FatalityVictimsCard({
+  crash,
+}: FatalityVictimsCardProps) {
+  const victims = crash.people_list_view?.filter(
+    (person) => person.prsn_injry_sev_id === 4
+  );
+  const units = crash.units;
+
+  return (
+    <Card>
+      <Card.Header>
+        <Card.Title>Victims</Card.Title>
+      </Card.Header>
+      <Card.Body className="pt-1">
+        <ListGroup variant="flush">
+          {units?.map((unit) => {
+            const unitVictims = victims?.filter(
+              (victim) => victim.unit_nbr === unit.unit_nbr
+            );
+            if (unitVictims && unitVictims?.length > 0) {
+              const unitYearMakeModel =
+                `${unit.veh_mod_year || ""} ${unit.veh_make?.label || ""} ${unit.veh_mod?.label || ""}`.trim();
+              return (
+                <ListGroupItem key={unit.id}>
+                  <div className="d-flex w-100 justify-content-start align-items-center">
+                    <h5 className="mb-1 me-2">
+                      <span className="fw-bold"> Unit {unit.unit_nbr}</span>
+                    </h5>
+                    <div className="d-flex flex-grow-1 justify-content-start text-secondary">
+                      <span>
+                        {unit.unit_desc?.label}
+                        {unitYearMakeModel ? ` | ${unitYearMakeModel}` : ""}
+                      </span>
+                    </div>
+                  </div>
+                  {unitVictims?.map((victim) => (
+                    <ListGroupItem key={victim.id} style={{ border: "none" }}>
+                      <div className="d-flex w-100 justify-content-between">
+                        <div className="d-flex align-items-center">
+                          <span className="fw-bold me-2">
+                            {victim.prsn_first_name} {victim.prsn_mid_name}{" "}
+                            {victim.prsn_last_name}
```

Comment:
would you add some conditional formatting to make sure we don't have empty spaces filling if any of the name parts are missing? I dunno if it makes sense to reuse [this formatter function](https://github.com/cityofaustin/vision-zero/blob/35f98c68c8615faf86ce22a97d3e5f8503038cb6/editor/configs/peopleColumns.tsx#L144).